src/effects/VST/VSTEffect.cpp

```diff
@@ -3599,4 +3597,42 @@ EffectSettings VSTEffect::MakeSettings() const
 }
 
 
+// Default, do-nothing implementations of virtuals in VSTEffectWrapper
+void VSTEffectWrapper::NeedIdle()
+{   
+}
+
+void VSTEffectWrapper::UpdateDisplay()
+{
+}
+
+VstTimeInfo* VSTEffectWrapper::GetTimeInfo()
+{
+   return nullptr;
+}
+
+void VSTEffectWrapper::SetBufferDelay(int samples)
+{
+}
+
+float VSTEffectWrapper::GetSampleRate()
+{
+   return 44100.0f;
+}
+
+int VSTEffectWrapper::GetProcessLevel()
+{
```

Comment:
This is the one virtual function of `VSTEffectWrapper` that we can define in the base and then not override again, or so we think now.

So the first commit might also remove `VSTEffect::GetProcessLevel()`